src/App.ts

```diff
@@ -241,6 +242,17 @@ export default class App {
 
   private hasCustomErrorHandler: boolean;
 
+  // used for the deferred initialization
```

Comment:
What about more direct reference to the relevant option?
```suggestion
  // used when deferInitialization is true
```